### PR TITLE
[NO-ISSUE] test: add e2e test to edge connectors

### PR DESCRIPTION
**Pull request by lucasmendes21**
### What is the new behavior introduced by this PR?
test: add e2e test to edge connectors

### Does this PR introduce breaking changes?

- [x] No
- [ ] Yes

### Does this PR introduce UI changes? Add a video or screenshots here.
NO
### Does it have a link on Figma?
NO
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari
- [x] Brave

### CODE REVIEW

File: cypress/e2mock/edge-connectors/create-edge-connectors.cy.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    cy.get('#type_2').click()
+    cy.get(edgeConnectors.edgeStorage.bucket)
+      .clear()
+      .type('advanced-filter-graphql')
+    cy.get(edgeConnectors.edgeStorage.prefix)
+      .clear()
+      .type('20241010173031')
+    cy.get(edgeConnectors.saveButton).click()
+
+    // // 3. Verifica o alias e validações UI sem backend
```
Comment (vinigfer) [inline cypress/e2mock/edge-connectors/create-edge-connectors.cy.js:121]: English only in the codebase

**Review by aloisio-m-bastian — APPROVED** [review]

**Review by peterpaulo-azion — APPROVED** [review]

**Review by vinigfer — APPROVED** [review]
